### PR TITLE
Use reference array

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

reformats yaml files where references are in a list, eg.

```
...
references:
- samm2:V-SD-3-A
- iso27001-2017:14.1
```

### CODE REVIEW

File: data-new/CultureAndOrganization/EducationAndGuidance.yaml [hunk 1]
```diff
@@ -114,9 +113,9 @@ Education and Guidance:
       resources: 1
     usefulness: 3
     level: 3
-    samm: IM-3, ST-3, SR2-B
-    iso27001-2017:
-    - 16.1.6
+    references:
+    - samm:IM-3, ST-3, SR2-B
```
Comment (wurstbrot) [inline data-new/CultureAndOrganization/EducationAndGuidance.yaml]: Should this become 
samm:IM-3
samm:ST-3
samm:SR2-B
?

File: data-new/TestAndVerification/Consolidation.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
       [dependabot](https://dependabot.com/)
     defect-dojo: &defect-dojo >-
       [OWASP Defect Dojo](https://github.com/DefectDojo/django-DefectDojo)
     purify: &purify https://github.com/faloker/purify
-    owasp-dependency-check: &owasp-dependency-check OWASP Dependency Check Jenkins Plugin
+    owasp-dependency-check: &owasp-dependency-check OWASP Dependency Check Jenkins
+      Plugin
     logparser-jenkins: &logparser-jenkins LogParser Jenkins Plugins
-    mozilla-zest: &mozilla-zest   https://github.com/mozilla/zest
+    mozilla-zest: &mozilla-zest https://github.com/mozilla/zest
```
Comment (wurstbrot) [inline data-new/TestAndVerification/Consolidation.yaml:14]: Is this intentional or should we remove the link?

**Review by wurstbrot — APPROVED** [review]

File: data-new/CultureAndOrganization/EducationAndGuidance.yaml (hunk 1 above)
Comment (ioggstream) [inline data-new/CultureAndOrganization/EducationAndGuidance.yaml]:
```suggestion
    - samm:IM-3
    - samm:ST-3
    - samm:SR2-B
```

Comment (ioggstream) [reply data-new/CultureAndOrganization/EducationAndGuidance.yaml]: yes!

File: data-new/TestAndVerification/Consolidation.yaml (hunk 2 above)
Comment (ioggstream) [reply data-new/TestAndVerification/Consolidation.yaml:14]: The link is ok.

**Pull request merged**